[2025-04-24T13:47:17.487+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-24T13:47:17.517+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_spark_connection.ping_spark manual__2025-04-24T13:41:40.213613+00:00 [queued]>
[2025-04-24T13:47:17.524+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_spark_connection.ping_spark manual__2025-04-24T13:41:40.213613+00:00 [queued]>
[2025-04-24T13:47:17.524+0000] {taskinstance.py:2306} INFO - Starting attempt 5 of 6
[2025-04-24T13:47:17.538+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): ping_spark> on 2025-04-24 13:41:40.213613+00:00
[2025-04-24T13:47:17.543+0000] {standard_task_runner.py:63} INFO - Started process 371 to run task
[2025-04-24T13:47:17.545+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'test_spark_connection', 'ping_spark', 'manual__2025-04-24T13:41:40.213613+00:00', '--job-id', '214', '--raw', '--subdir', 'DAGS_FOLDER/test_spark_connection.py', '--cfg-path', '/tmp/tmpy5bi_vcl']
[2025-04-24T13:47:17.547+0000] {standard_task_runner.py:91} INFO - Job 214: Subtask ping_spark
[2025-04-24T13:47:17.586+0000] {task_command.py:426} INFO - Running <TaskInstance: test_spark_connection.ping_spark manual__2025-04-24T13:41:40.213613+00:00 [running]> on host 445358847ece
[2025-04-24T13:47:17.728+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test_spark_connection' AIRFLOW_CTX_TASK_ID='ping_spark' AIRFLOW_CTX_EXECUTION_DATE='2025-04-24T13:41:40.213613+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-24T13:41:40.213613+00:00'
[2025-04-24T13:47:17.728+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-24T13:47:17.746+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-24T13:47:17.747+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'curl -s http://localhost:7077/json || echo "Spark not reachable"']
[2025-04-24T13:47:17.759+0000] {subprocess.py:86} INFO - Output:
[2025-04-24T13:47:17.772+0000] {subprocess.py:93} INFO - Spark not reachable
[2025-04-24T13:47:17.772+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-24T13:47:17.773+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-24T13:47:17.790+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_spark_connection, task_id=ping_spark, run_id=manual__2025-04-24T13:41:40.213613+00:00, execution_date=20250424T134140, start_date=20250424T134717, end_date=20250424T134717
[2025-04-24T13:47:17.837+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-24T13:47:17.857+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-24T13:47:17.860+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
